CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# Usage (Linux):
+#   $ cd OpenTombSrc
+#   $ mkdir build && cd build
+#   $ cmake .. && make
+
+cmake_minimum_required(VERSION 2.8)
+
+project(OpenTomb)
+
+add_subdirectory(doc)
```

Comment:
Please add something like  `message(WARN "This CMakeLists.txt does not generate executables, it is only used to process the documentation.")`. (Code not tested, I'm not sure about the `WARN` keyword.)